blog-pipeline/src/pipeline_smoke_test.py

```
@@ -706,6 +706,99 @@ def run_checks():
+        try:
+            import requests as _req
+            resp = _req.get("https://api.dataforseo.com/v3/appendix/user_data",
+                auth=(dforseo_login, dforseo_pass), timeout=10)
+            if resp.status_code == 200:
+                balance = resp.json().get("tasks", [{}])[0].get("result", [{}])[0].get("money", {}).get("balance", 0)
+                results.append(CheckResult("DataForSEO balance", "warning", balance >= 1.0,
+                    f"${balance:.2f}" if balance >= 1.0 else f"LOW: ${balance:.2f}"))
+            else:
+                results.append(CheckResult("DataForSEO balance", "warning", False,
+                    f"API status={resp.status_code}"))
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)